dts/arm/silabs/siwg917.dtsi

```diff
@@ -24,6 +24,19 @@
 		cpu0: cpu@0 {
 			compatible = "arm,cortex-m4f";
 			reg = <0>;
+			cpu-power-states = <&pstate_ps4_standby &pstate_ps4_sleep>;
```

Comment:
Should the sleep mode be skipped until all drivers implement power management callbacks properly? The state could still be defined in DTS, just not registered for use by the CPU.